src/pages/Subjects.tsx

```diff
@@ -0,0 +1,60 @@
+import { useState, useEffect, ReactNode } from "react";
+import Layout from "../components/Layout/Layout";
+interface Subject {
+  [x: string]: ReactNode;
+  id: number;
+  name: string;
+  description: string;
+}
```

Comment:
Can this be reused? Either this week or next, a stretch goal might be building out an interface library.